llvm/lib/Target/X86/X86FlagsCopyLowering.cpp

```diff
@@ -806,7 +811,8 @@ void X86FlagsCopyLoweringPass::rewriteArithmetic(
   // Insert an instruction that will set the flag back to the desired value.
   Register TmpReg = MRI->createVirtualRegister(PromoteRC);
   auto AddI =
-      BuildMI(MBB, MI.getIterator(), MI.getDebugLoc(), TII->get(X86::ADD8ri))
+      BuildMI(MBB, MI.getIterator(), MI.getDebugLoc(),
+              TII->get(Subtarget->hasNDD() ? X86::ADD8ri_ND : X86::ADD8ri))
```

Comment:
Do we need to use macro like GET_NDD_IF_ENABLED(OPC_ND, OPC).

Reply:
We may need to define it in other files. But in this file, only one use.